ui/core/src/users/RelayQueryConfig.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * file: users/RelayQueryConfig.js
+ *
+ * Exports a RelayQueryConfig to be used with users/index.js
+ *
+ */
+import Relay from 'react-relay';
+
+const NodeQueries = {
```

Comment:
Please move both of this queries **shared** dir. Both `viewer` and `node` are going to be used in many modules